src/Root.sol

```diff
@@ -124,7 +124,7 @@ contract Root is Auth, IRoot {
         } else if (call == MessagesLib.Call.RecoverTokens) {
             (address target, address token, address to, uint256 amount) =
                 (message.toAddress(1), message.toAddress(33), message.toAddress(65), message.toUint256(97));
-            RecoverLike(target).recoverTokens(token, to, amount);
+            recoverTokens(target, token, to, amount);
```

Comment:
Unrelated to the PR but this was off (no reason to have the internal method otherwise)